TranquilizerGun/EventsHandler.cs

```diff
@@ -86,7 +86,7 @@ public class EventsHandler {
 
         public void HurtEvent(HurtingEventArgs ev) {
             try {
-                if(ev.Attacker == null || ev.Attacker == ev.Target || plugin.Config.roleBlacklist.Contains(ev.Target.Role))
+                if((plugin.Config.ClassDScientistsOnly && ev.Target.Role != RoleType.ClassD && ev.Target.Role != RoleType.Scientist) || ev.Attacker == null || ev.Attacker == ev.Target || plugin.Config.roleBlacklist.Contains(ev.Target.Role))
```

Comment:
This if basically says:
if config says it should be class-d or scientist only && role isnt classD && role isnt scientist
so if im class d, this triggers because im not a scientist and if im a scientist this triggers because im not a class d